pkg/eestream/rs_test.go

```diff
@@ -6,6 +6,7 @@ package eestream
 import (
 	"bytes"
 	"context"
+	cryptorand "crypto/rand"
```

Comment:
in tests you can use `math/rand`.

`rand.New(rand.NewSource(rand.Int63()))` <-- to get a reader from `math/rand`. Not sure whether there's an easier way.

Reply:
Good to know how to create a random reader from `math/rand`. Thanks!